packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -272,6 +274,55 @@ async function approveWeth() {
   await tx.wait()
 }
 
+async function approveErc20(l1ERC20Token: Contract) {
+  console.log('Approving ERC20...')
+  const erc20Bridger = await Erc20Bridger.fromProvider(childProvider)
+  const approvalTx = await erc20Bridger.approveToken({
+    erc20ParentAddress: l1ERC20Token.address,
+    parentSigner: userWallet.connect(parentProvider),
+    amount: parseEther('1') // only approve 1 token for deposits, later we will need MAX amount approval during approve tests
+  })
+  await approvalTx.wait()
+}
+
+async function fundErc20ToParentChain(l1ERC20Token: Contract) {
+  console.log('Funding ERC20 on Parent Chain...')
+  // Send deployed ERC-20 to the test userWallet
+  const transferTx = await l1ERC20Token
+    .connect(localWallet.connect(parentProvider))
+    .transfer(await userWallet.getAddress(), parseEther('5'))
+  await transferTx.wait()
+}
+
+async function fundErc20ToChildChain(l1ERC20Token: Contract) {
+  console.log('Funding ERC20 on ChildChain...')
+  // first deploy the ERC20 to L2 (if not, it might throw a gas error later)
+  await deployERC20ToChildChain(l1ERC20Token.address)
+  const erc20Bridger = await Erc20Bridger.fromProvider(childProvider)
+  const ethSigner = localWallet.connect(parentProvider)
```

Comment:
why is this called `ethSigner`? should be `parentSigner`?

Reply:
Yeah the renaming PR merged during development so this was a miss. Ty!